### PR TITLE
Total stream USD value

### PR DESCRIPTION
**Pull request by elvijsTDL**
Screenshot 2021-12-23 at 01 42 38

### CODE REVIEW

**Comment by mikeghen** [comment]
Can you have the USD value up top in black and blue, then in smaller font size under have the TVS in the native token, sort of like how its done here
Screen Shot 2021-12-23 at 10 06 59 AM

**Review by mikeghen — CHANGES_REQUESTED** [review]
Nice work otherwise

**Comment by elvijsTDL** [comment]
Done @mikeghen 

Screenshot 2021-12-24 at 04 18 00
Screenshot 2021-12-24 at 04 19 03

File: src/components/layout/PanelChange/index.tsx [hunk 1]
```diff
@@ -63,6 +65,14 @@ export const PanelChange: FC<IProps> = ({
     setIsLoading(mainLoading);
   }, [mainLoading]);
 
+  function getFormattedNumber(num: string) {
+    return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  }
+
+  function getFlowUSDValue(flow:string) {
+    return (parseFloat(flow as string) * coingeckoPrice).toFixed(2);
```
Comment (mikeghen) [inline src/components/layout/PanelChange/index.tsx:73]: if we arent going to do the K,M abbreviations, can we at least not show the decimal cents? Here, one has no cents after the decimal, another has just 1 number after the decimal, and another has 2 numbers after the decimal
Screen Shot 2021-12-25 at 9 28 16 AM

**Review by mikeghen — CHANGES_REQUESTED** [review]
just can we remove the decimals

**Comment by elvijsTDL** [comment]
@mikeghen Removed the cents from the USD price, left them at token amounts, because they add up to a bunch of value for tokens like ETH
Screenshot 2021-12-26 at 18 11 21

**Pull request merged**